src/rgw/rgw_sts.h

```diff
@@ -147,6 +192,7 @@ class Credentials {
 //AssumedRoleUser, Credentials, PackedpolicySize
 using AssumeRoleResponse = std::tuple<int, AssumedRoleUser, Credentials, uint64_t> ;
 using GetSessionTokenResponse = std::tuple<int, Credentials>;
+using AssumeRoleWithWebIdentityResponse = std::tuple<int, AssumedRoleUser, string, Credentials, uint64_t, string, string>;
```

Comment:
this response is getting big enough to consider using a struct instead of tuple. that way we can give names to the different strings and avoid confusion

Reply:
not sure, unless we're going to change the related responses as a group?